modules/urideploy/pom.xml

```diff
@@ -111,13 +111,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
Removing ignite-tools may fail the suite since ignite-tools is needed to assist their execution. Why?

Reply:
I removed only duplicated dependency definition. I mean that there is a two line-to-line copies of 

        <dependency>
            <groupId>org.apache.ignite</groupId>
            <artifactId>ignite-tools</artifactId>
            <version>${project.version}</version>
            <scope>test</scope>
        </dependency>

one right after another. (Expand change view up for several lines)